tests/org.eclipse.n4js.transpiler.es5.tests/testdata/annotations/IDEBUG_246_DescriptionWithSingleQuote.n4js.xt

```diff
@@ -26,13 +26,17 @@ stderr:
 @Description("<'>")
 class A{}
 
+// Remark: avoid using x.toString() calls since different node versions emit different string literals!
+
 console.log(A)
```

Comment:
But this is still using `x.toString()`, isn't it?
Does not seem to cause issues (between the node versions we and you are using, atm) but still having this line in place makes the newly added comment a bit confusing.

(same in the other files starting with `IDEBUG_246_*` in this folder)

Reply:
To be more accurate: toString() on Objects is a problem, not on Strings

Reply:
But `A` in line `console.log(A);` evaluates to a class constructor, not a string. Or am I overlooking something?